### ISSUE TITLE
(doc) RTD fails to generate taurus.qt.qtgui API docs

### ISSUE THREAD

**Issue by emiliosproyects** [taurus-org/taurus#490]
Hello, 
Link broken in the web site. See the issue in the next link:

http://www.taurus-scada.org/en/latest/devel/api/taurus/qt/qtgui.html

Access to the API doc's in 'latest ' doc version is broken.

**Comment by cpascual** [comment]
The problem is likely to be related with the early import of qtpy in #473 
RTD does not have the mock for qtpy and this is causing errors that result in the API docs for  taurus.qt.qtgui not being generated.

If this is confirmed, I see the following approaches (in increasing order of effort needed):
- a) As a very ugly (emergency) workaround, we could do the early import only if we are not in an RTD build.
- b) A slightly less ugly workaround is to insert a simple mock of qtpy in  `doc/source/conf.py`
- c) We could also update `doc/buildmock.py` to support qtpy and regenerate `doc/mock.zip` 
- d) The "proper" solution would be to fix #408 

Since we are in a hurry (this should be fixed before Jul17) I personally would go for a) or b) and then straight to d)

**Comment by cpascual** [comment]
The solution I proposed does not work. I am preparing a better solution that will also fix fix #267 and other RTD failures.

**Comment by cpascual** [comment]
Note: I tested this using an empty virtualenv where I declared the READTHEDOCS envvar and run sphinx:

```
$ mkvirtualenv rtd
(rtd) $ export READTHEDOCS='True'
(rtd) $ pip install sphinx
(rtd) $ pip install sphinx_rtd_theme
(rtd) $ pip install mock
(rtd) $ sphinx-build -T -E -b html -d _build/doctrees-readthedocs -D language=en doc/source/ _build/html
```
